### PR TITLE
docs(devtools): add docs sdk js

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  paragraph: This page explains how to install and use the Scaleway JS software developement kit (SDK)
+tags: devtools sdk go-sdk scaleway-sdk go golang
+dates:
+  validation: 2023-11-14
+  posted: 2021-11-14
+categories:
+  - devtools
+---
+
+The Scaleway JS SDK, developed by Scaleway, is a dynamic tool for developers who use JavaScript to interact with Scaleway's cloud services. It makes complex tasks like managing servers, handling storage operations, and setting up networking more manageable by automating intricate API calls and the authentication process. This SDK is instrumental for integrating or programmatically managing Scaleway's cloud resources.
```
Comment (nerda-codes) [inline developer-tools/scaleway-sdk/js-sdk.mdx:16]:
```suggestion
The Scaleway JS SDK is a dynamic tool for developers who use JavaScript to interact with Scaleway's cloud services. It makes complex tasks like managing servers, handling storage operations, and setting up networking more manageable by automating intricate API calls and the authentication process. This SDK is instrumental for integrating or programmatically managing Scaleway's cloud resources.
```

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+<Message type="important">
+  The SDK is designed to function in a Node.js environment. Browser-based requests are not supported due to CORS restrictions.
+</Message>
+
+## Installing the JS SDK
+
+To use the SDK, you need an access key and a secret key for the Scaleway API. Refer to Scaleway's [documentation on API keys](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-api-keys/) for instructions on creating and retrieving these keys.
```
Comment (nerda-codes) [inline developer-tools/scaleway-sdk/js-sdk.mdx:24]:
```suggestion
To use the SDK, you need an access key and a secret key for the Scaleway API. Refer to Scaleway's [documentation on API keys](/identity-and-access-management/iam/how-to/create-api-keys/) for instructions on creating and retrieving these keys.
```

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Here's a basic setup example:
```
Comment (nerda-codes) [inline developer-tools/scaleway-sdk/js-sdk.mdx:26]:
```suggestion
Below is a basic setup example:
```

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+const api = new Registry.v1.API(client)
+```
+
+## Using the SDK
+
+The SDK is written in TypeScript, offering the full benefits of this language. 
+
+1. For any product, its types are available under `Product.version`. For example, access the `Image` interface of Registry v1 using `Registry.v1.Image`.
+
+2. We also provide exports of **errors** under the `Errors` namespace, which helps in differentiating various error cases, like `Errors.PermissionsDeniedError`.
```
Comment (nerda-codes) [inline developer-tools/scaleway-sdk/js-sdk.mdx:48]:
```suggestion
2. Scaleway also provides exports of **errors** under the `Errors` namespace, which helps differentiating various error cases, like `Errors.PermissionsDeniedError`.
```

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+## Contributing
+
+This project is in the early stages of development and is actively evolving. If you're interested in contributing, join us at [#opensource](https://scaleway-community.slack.com/app_redirect?channel=opensource) in Scaleway's Slack community for discussions.
```
Comment (nerda-codes) [inline developer-tools/scaleway-sdk/js-sdk.mdx:52]:
```suggestion
This project is in the early stages of development and is actively evolving. If you are interested in contributing, join us at [#opensource](https://scaleway-community.slack.com/app_redirect?channel=opensource) in Scaleway's Slack community for discussions.
```

**Review by nerda-codes — APPROVED** [review]

File: developer-tools/scaleway-sdk/js-sdk.mdx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+})
+
+const api = new Registry.v1.API(client)
+```
+
+## Using the SDK
+
+The SDK is written in TypeScript, offering the full benefits of this language. 
+
+1. For any product, its types are available under `Product.version`. For example, access the `Image` interface of Registry v1 using `Registry.v1.Image`.
```
Comment (ldecarvalho-doc) [inline developer-tools/scaleway-sdk/js-sdk.mdx:46]: these do not seem to be steps, maybe put in a bullet list?

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request closed without merging**